src/core/DataHelper.ttslua

```diff
@@ -858,476 +941,704 @@ PLAYER_CARD_TOKEN_OFFSETS = {
 
 modeData = {
```

Comment:
Please have separate PRs. These two changes seem unrelated

Reply:
Once again the auto-formatting reformatted this...
Fixed it and now it should be clear why it's here 😅